guides/source/action_mailer_basics.md

```diff
@@ -851,18 +851,18 @@ Add this to your `config/environments/$RAILS_ENV.rb` file to send via Gmail:
 ```ruby
 config.action_mailer.delivery_method = :smtp
 config.action_mailer.smtp_settings = {
-  address:              'smtp.gmail.com',
-  port:                 587,
-  domain:               'example.com',
-  user_name:            '<username>',
-  password:             '<password>',
-  authentication:       'plain',
-  enable_starttls_auto: true,
-  open_timeout:         5,
-  read_timeout:         5 }
-```
-
-NOTE: On July 15, 2014, Google increased [its security measures](https://support.google.com/accounts/answer/6010255) to block attempts from apps it deems less secure.
+  address:         'smtp.gmail.com',
+  port:            587,
+  domain:          'example.com',
+  user_name:       '<username>',
+  password:        '<password>',
+  authentication:  'plain',
+  enable_starttls: true,
```

Comment:
This option isn't documented here https://github.com/rails/rails/blob/603631b335af8314593ff9995dea9e068b96cd8d/actionmailer/lib/action_mailer/base.rb#L405

Should it be?

Reply:
I just realised that `enable_starttls` was added in [mail version 2.7.0](https://github.com/mikel/mail/blob/2-7-stable/CHANGELOG.rdoc#label-Version+2.7.0+-282017-10-31-29). Surprisingly, the auto-feature was added first.

Action Mailer currently requires [version 2.5.4](https://github.com/rails/rails/blob/main/actionmailer/actionmailer.gemspec#L40).

I don't think there is any reason to bump the requirement just because of the guide, but the new option definitely deserves to be mentioned. I'll fix.

Reply:
Arg, I made this fix a long time ago but I somehow failed to push my branch. In the meantime this option has been documented in #44096.

Reply:
So if you are using an older version of `mail` and set this option, will that be fine?

Reply:
I have added a note about this.